requirements.txt

```diff
@@ -1,110 +1,124 @@
-astroid==1.3.8
-Babel==2.1.1
-CherryPy==3.8.0
-cliff==1.15.0
+appdirs==1.4.0
+astroid==1.4.5
+Babel==2.3.4
+backports.shutil-get-terminal-size==1.0.0
+chardet==2.3.0
+CherryPy==3.8.2
+cliff==2.0.0
 cmd2==0.6.8
+colorama==0.3.7
 cookies==2.2.1
-coverage==4.0
-debtcollector==0.8.0
-decorator==4.0.4
+coverage==4.0.3
+debtcollector==1.3.0
+decorator==4.0.9
 dj-static==0.0.6
-Django==1.8.5
-django-angular==0.7.15
-django-appconf==1.0.1
-django-compressor==1.5
-django-debug-toolbar==1.3.2
+Django==1.9.5
+django-angular==0.8.1
+django-appconf==1.0.2
+django-compressor==2.0
+django-debug-toolbar==1.4
 django-environ==0.4.0
-django-extensions==1.5.7
-django-filter==0.11.0
-django-grappelli==2.7.1
-django-libsass==0.4
-django-redis==4.2.0
+django-extensions==1.6.1
+django-filter==0.13.0
+django-grappelli==2.8.1
+django-libsass==0.6
+django-redis==4.4.2
 django-zurb-foundation==5.5.0
-djangorestframework==3.2.4
-dodgy==0.1.7
-factory-boy==2.5.2
+djangorestframework==3.3.3
+dodgy==0.1.9
+factory-boy==2.7.0
+fake-factory==0.5.7
 Flask==0.10.1
-freezegun==0.3.5
+freezegun==0.3.7
+futures==3.0.5
 gitdb==0.6.4
-GitPython==1.0.1
+GitPython==2.0.0
 glob2==0.4.1
-gunicorn==19.3.0
-honcho==0.6.6
-huey==0.4.9
-ipdb==0.8.1
-ipython==4.0.0
+gunicorn==19.4.5
+honcho==0.7.1
+huey==1.1.1
+ipdb==0.9.3
+ipython==4.2.0
 ipython-genutils==0.1.0
-iso8601==0.1.10
+iso8601==0.1.11
 itsdangerous==0.24
-jasmine==2.3.0
-jasmine-core==2.3.4
+jasmine==2.4.0
+jasmine-core==2.4.0
 Jinja2==2.8
-jira==0.50
-libsass==0.8.3
-logilab-common==1.0.2
-Markdown==2.6.2
+jira==1.0.3
+keystoneauth1==2.6.0
+lazy-object-proxy==1.2.2
+libsass==0.11.1
+logilab-common==1.2.0
+Markdown==2.6.6
 MarkupSafe==0.23
-mccabe==0.3.1
-mock==1.3.0
-monotonic==0.3
-msgpack-python==0.4.6
-mysqlclient==1.3.6
+mccabe==0.4.0
+mock==2.0.0
```

Comment:
@antoviaque Since we are using Python >= 3.3 I'm wondering if there is a special reason for keeping this as an external dependency (instead of using the built-in `unittest.mock`)?